src/main/java/Duke.java

```diff
@@ -1,10 +1,73 @@
-public class Duke {
+import java.util.Scanner;
+import java.util.*;
+
+public class
+Duke {
+    public static boolean containsNumbers(String string) {
+        if (string == null || string.isEmpty()) {
+            return false;
+        }
+        for (int i = 0; i < string.length(); ++i) {
+            if (Character.isDigit(string.charAt(i))) {
+                return true;
+            }
+        }
+        return false;
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        String greeting = ("____________________________________________________________\n"
+                + "Hello! I'm Duke\n" + "What can I do for you?\n"
+                + "____________________________________________________________\n"
+        );
+
+        String goodBye = ("____________________________________________________________\n"
+                + "Bye. Hope to see you again soon!\n"
+                + "____________________________________________________________\n");
```

Comment:
Perhaps you can change your constant names to be all uppercase using underscore to separate words